README.md

```diff
@@ -0,0 +1,7 @@
+# bases
```

Comment:
the other heading is capitalized, do you want this to be too?

Reply:
it's kind of stylized as lowercase but it doesn't really matter either way

Reply:
👍